### PR TITLE
Function calling improvements

### PR DESCRIPTION
**Pull request by Giom-V**

### CODE REVIEW

File: quickstarts/Function_calling.ipynb [hunk 1]
```diff
@@ -64,6 +64,17 @@
         "This notebook provides code examples to help you get started."
```
Comment (markmcd) [inline quickstarts/Function_calling.ipynb:64]: <p>Nit: Is this meant to be a code cell? (The multiple <code>#</code> at the start makes me think it's intended to be markdown)</p><p> </p><p>I would either merge this with the code cell below or turn it into a MD cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by markmcd — DISMISSED** [review]

File: quickstarts/Function_calling.ipynb (hunk 1 above)
Comment (Giom-V) [reply quickstarts/Function_calling.ipynb:64]: <p>Good catch, it was indeed meant to be markdown</p>

File: quickstarts/Function_calling.ipynb [hunk 2]
```diff
@@ -48,7 +48,7 @@
       "source": [
         "<table align=\"left\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb\"><img src=\"../images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb\"><img src=\"https://github.com/Giom-V/gemini-api-cookbook/blob/function_calling/images/colab_logo_32px.png?raw=1\" />Run in Google Colab</a>\n",
```
Comment (markmcd) [inline quickstarts/Function_calling.ipynb:51]: Use relative links here. We don't to break this guide if/when you delete your branch (and relative links should work across forks too)

File: quickstarts/Function_calling.ipynb [hunk 3]
```diff
@@ -749,7 +824,17 @@
...
-        "- If you reply manually using [chat.send_message](https://ai.google.dev/api/python/google/generativeai/ChatSession#send_message) or [model.generate_content](https://ai.google.dev/api/python/google/generativeai/GenerativeModel#generate_content) remember thart the API is stateless you have to send the whole conversation history (a list of [content](https://ai.google.dev/api/python/google/generativeai/protos/Content) objects), not just the last one containing the `FunctionResponse`."
+        "- If you reply manually using [chat.send_message](https://ai.google.dev/api/python/google/generativeai/ChatSession#send_message) or [model.generate_content](https://ai.google.dev/api/python/google/generativeai/GenerativeModel#generate_content) remember thart the API is stateless you have to send the whole conversation history (a list of [content](https://ai.google.dev/api/python/google/generativeai/protos/Content) objects), not just the last one containing the `FunctionResponse`.\n",
```
Comment (markmcd) [inline quickstarts/Function_calling.ipynb:827]: These API links don't work any more. You don't have to change the old ones, but don't add new links that don't work.

(by "don't work", I mean that we redirect anything from `/api/python/*` to the same page - a better experience would be to provide a link into the [specific pages on GitHub](https://github.com/google-gemini/generative-ai-python/blob/main/docs/api/google/generativeai/protos/Content.md), or to the [API reference](https://ai.google.dev/api/generate-content#request-body) for non-Python-specifics)

**Review by markmcd — APPROVED** [review]

File: quickstarts/Function_calling.ipynb (hunk 3 above)
Comment (Giom-V) [reply quickstarts/Function_calling.ipynb:827]: Shoud I change the links to https://github.com/google-gemini/generative-ai-python/blob/main/docs/api/google/generativeai/ChatSession.md#send_message for example?

Comment (markmcd) [reply quickstarts/Function_calling.ipynb:827]: Yeah that works. I prefer using the website where we can but for specific symbols that makes sense.

**Review by markmcd — APPROVED** [review]

**Review by markmcd — APPROVED** [review]

**Pull request merged**